buildSrc/linux.gradle

```diff
@@ -120,28 +119,25 @@ setupTools("linux_gtk3",
         exec {
             commandLine("${toolchainDir}pkg-config", "--cflags", "gtk+-3.0", "gthread-2.0", "xtst")
             setStandardOutput(results2);
-            ignoreExitValue(true)
         }
         propFile << "cflagsGTK3=" << results2.toString().trim() << "\n";
 
         ByteArrayOutputStream results4 = new ByteArrayOutputStream();
         exec {
             commandLine("${toolchainDir}pkg-config", "--libs", "gtk+-3.0", "gthread-2.0", "xtst")
             setStandardOutput(results4);
-            ignoreExitValue(true)
         }
         propFile << "libsGTK3=" << results4.toString().trim()  << "\n";
 
     },
     { properties ->
-        def ccflags =  properties.getProperty("cflagsGTK3")
-        def ldflags =  properties.getProperty("libsGTK3")
-        if (ccflags && ldflags) {
-            gtk3CCFlags.addAll(ccflags.split(" "))
-            gtk3LinkFlags.addAll(ldflags.split(" "))
+        def cflagsGTK3 = properties.getProperty("cflagsGTK3")
+        def libsGTK3 = properties.getProperty("libsGTK3")
+        if (cflagsGTK3 && libsGTK3) {
+            gtk3CCFlags.addAll(cflagsGTK3.split(" "))
+            gtk3LinkFlags.addAll(libsGTK3.split(" "))
```

Comment:
Here only variable names are changed to make them similar to the names used for gtk2 code.